### DISCUSSION TITLE
Output does not work with Polly

### DISCUSSION THREAD

**Discussion by holanino** [ThioJoe/SRT-To-SSML#6] [General]
Anyone have a solution for getting an SRT file and converting it to AWS Polly?

**Comment by ThioJoe** [comment]
Can you be a bit more specific about the issue?

First it's important to realize that Amazon Polly only supports the duration feature for non-neural voices.

~~Also, were you sure to change the `durationAttributeName` value to `amazon:max-duration` ?~~

The latest version of the script now automatically sets those variables if you set `serviceMode = "amazon-standard-voice"`

**Reply by holanino** [reply]
Thank you that worked!
